pulser/sequence.py

```diff
@@ -543,7 +563,7 @@ def align(self, *channels):
             if delta > 0:
                 self._delay(delta, id)
 
-    def build(self, **vars):
+    def build(self, **vars: Any) -> Sequence:
```

Comment:
Actually for the `vars`, you can specify them to be: `Union[int, float, str, ArrayLike]`